Add docs for reapply_sysctl TuneD functionality (#317)
openshift/cluster-node-tuning-operator: README.md

management interface to users of node-level sysctls and more
flexibility to add [custom tuning](#custom-tuning-specification)
specified by user needs. The Operator manages the containerized
[TuneD](https://github.com/redhat-performance/tuned/)
daemon for [OpenShift](https://openshift.io/) as
a Kubernetes DaemonSet. It ensures [custom tuning
specification](#custom-tuning-specification) is passed to all
containerized TuneD daemons running in the cluster in the format
that the daemons understand. The daemons run on all nodes in the
cluster, one per node.

> Node-level settings applied by the containerized TuneD daemon are rolled back
> on an event that triggers a profile change or when the containerized TuneD
> daemon is terminated gracefully by receiving and handling a termination signal.

Reviewer: This sentence is a little bit confusing to me. Will settings be rolled back when a profile is changed, or will they just updated to the settings in the latest profile?

Depending on the intended meaning, perhaps a better wording could be:
```suggestion
The containerized TuneD daemon will roll-back the custom node-level settings when a profile is changed or when the container receives a termination signal.
```

Author: > This sentence is a little bit confusing to me. Will settings be rolled back when a profile is changed, or will they just updated to the settings in the latest profile?

They will always be rolled back prior to applying a new profile.
